### ISSUE TITLE
Open Source XeSS

### ISSUE THREAD

**Issue by MOVZX** [intel/xess#5]
Edit: I get it now XeSS wasn't meant to be open sourced right?

**Comment by sunzhuoshi** [comment]
Yeah, still not open source yet.

**Comment by trsh** [comment]
But why?

**Comment by sobkas** [comment]
> > But why?
> 
> Sorry, I can't answer this question for limited information.
I'm sorry but this problem can be solved by asking someone who knows this.
This the whole point of not being solitary developer...

**Comment by bezirg** [comment]
There are Intel employees on Twitter, even at this moment, claiming that this closed-source binary code is actually open-source; just because somebody had this idea to put .exe files inside a git repository and upload it to github.

Please fix this. This is getting out of hand.

**Comment by tr1p0p** [comment]
Yeah, can you actually do what you... Tell to everyone you will do ?

Or maybe just read the definition of "open-source" before putting everywhere on website and news.

**Comment by dsge** [comment]
You can use an open-source and cross-platform web browser to download the closed-source windows-only binaries from Github.

**Comment by fatboychummy** [comment]
> No reverse engineering, decompilation, or disassembly of this Software is permitted

From the [license](https://github.com/intel/xess/blob/0d32d340598c87bba6f632127231a47e9412157d/licenses/LICENSE.pdf).

And Microsoft people are running around claiming it is open sourced. Lol.

**Comment by Okoyl** [comment]
![image](https://github.com/intel/xess/assets/3100771/298b9c98-73b1-4b4c-a751-da974a749278)

This is funny, this repo has a license in PDF format instead of markdown. Oh and the horror of exe and dll files in a git repo.

**Comment by relic664** [comment]
@xess-intel @rdower 

It's now 2025 and more than 2 years since this issue was opened. We've now seen the announcement of XeSS 2, yet still no updates about the actual nature of "open-source" and Linux support (still relying on mesa hacks to even get Arc working in some instances #22).

I think the community and also your customers of the Arc series GPUs deserve some answers about XeSS and it's availability and support in Linux.

**Comment by Eonfge** [comment]
I must inform you that Intel (NASDAQ: INTC) is engaging in false advertising and deceitful market practices.

Both UK and US courts have affirmed that calling a product "Open Source" without meeting the OSI criteria is deceitful.

- https://www.theregister.com/2022/03/17/court_open_source/
- https://opensource.org/blog/court-affirms-its-false-advertising-to-claim-software-is-open-source-when-its-not

I would therefore recommend that Legal gets involved and that you withdraw and formal and informal statement. If not, you'll risk legal actions.

---

Any US or UK citizens here, who are fed up with Intel's doublespeak, can use the legal cases in the news articles as basis for a formal complaint to your market regulator. Since there is already case law, it wl will be easier for these organisations to send a formal letter.

**Comment by DealsBeam** [comment]
> I must inform you that Intel (NASDAQ: INTC) is engaging in false advertising and deceitful market practices.
> 
> Both UK and US courts have affirmed that calling a product "Open Source" without meeting the OSI criteria is deceitful.
> 
> * [theregister.com/2022/03/17/court_open_source](https://www.theregister.com/2022/03/17/court_open_source/)
> * [opensource.org/blog/court-affirms-its-false-advertising-to-claim-software-is-open-source-when-its-not](https://opensource.org/blog/court-affirms-its-false-advertising-to-claim-software-is-open-source-when-its-not)
> 
> I would therefore recommend that Legal gets involved and that you withdraw and formal and informal statement. If not, you'll risk legal actions.
> 
> Any US or UK citizens here, who are fed up with Intel's doublespeak, can use the legal cases in the news articles as basis for a formal complaint to your market regulator. Since there is already case law, it wl will be easier for these organisations to send a formal letter.

Surely there are more important things in life to care about than this

**Comment by tr1p0p** [comment]
> > I must inform you that Intel (NASDAQ: INTC) is engaging in false advertising and deceitful market practices.
> > Both UK and US courts have affirmed that calling a product "Open Source" without meeting the OSI criteria is deceitful.
> > 
> > * [theregister.com/2022/03/17/court_open_source](https://www.theregister.com/2022/03/17/court_open_source/)
> > * [opensource.org/blog/court-affirms-its-false-advertising-to-claim-software-is-open-source-when-its-not](https://opensource.org/blog/court-affirms-its-false-advertising-to-claim-software-is-open-source-when-its-not)
> > 
> > I would therefore recommend that Legal gets involved and that you withdraw and formal and informal statement. If not, you'll risk legal actions.
> > Any US or UK citizens here, who are fed up with Intel's doublespeak, can use the legal cases in the news articles as basis for a formal complaint to your market regulator. Since there is already case law, it wl will be easier for these organisations to send a formal letter.
> 
> Surely there are more important things in life to care about than this

...Why you post here that there's better thing tondo than... Post here ?

Go away then ?

**Comment by MarcHarmening** [comment]
Dear Intel Graphics Team,

As a Linux user and potential customer, I’m deeply disappointed to see that Intel XeSS is not officially supported on Linux, despite being advertised as a cross-platform solution.

For many of us, XeSS was a major selling point of Intel Arc GPUs — offering performance close to DLSS and better quality than FSR. However, the current state of XeSS under Linux is either completely unsupported or limited to fallback modes, with no access to XMX acceleration or proper driver/API integration.

I urge you to:

    Provide official XeSS support for Linux, including Proton/Wine compatibility.

    Open up the necessary APIs (XMX, SYCL, etc.) to allow proper implementation.

    Communicate transparently about your roadmap for Linux support.

Without XeSS, I — like many in the Linux gaming community — will likely reconsider purchasing Intel GPUs and stick with vendors that offer full feature parity across platforms.

Sincerely,

Marc

**Comment by wancoder** [comment]
XeSS still only open source in machine code/assembly?
Aww, shucks, time to lift that rock and pump some machine code.
